intellij/src/saros/intellij/ui/actions/ConnectServerAction.java

```diff
@@ -34,15 +35,22 @@ public String getActionName() {
 
   /** Connects with the given user. */
   public void executeWithUser(String user) {
-    XMPPAccount account = accountStore.findAccount(user);
-    accountStore.setAccountActive(account);
+    JID jid = new JID(user);
+    XMPPAccount account = accountStore.getAccount(jid.getName(), jid.getDomain());
+
+    if (account == null) return;
+
+    accountStore.setDefaultAccount(account);
```

Comment:
I do not know when this called ? Is the "user" fetched from the store or how is the data obtained. If it from the store you just mimic the Eclipse behavior aka if you connect with another account this one will become the default account.

Reply:
Ok found it. Same behavior as Eclipse. Just copied or intended ?